notebooks/official/automl/sdk_automl_tabular_regression_batch_bq.ipynb

```diff
@@ -0,0 +1,969 @@
+{
```

Comment:
<p>Vertex AI SDK for Python instead of Vertex SDK</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/331/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Done.